org.citra.citra.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.citra.citra",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "citra-qt",
+    "rename-desktop-file": "citra.desktop",
+    "rename-icon": "citra",
+    "rename-appdata-file": "org.citra.citra.appdata.xml",
+    "build-options": {
+        "env": {
+            "CI": "1",
+            "GITHUB_ACTIONS": "1",
+            "GITHUB_REPOSITORY": "citra-emu/citra",
+            "GIT_TAG_NAME": "nightly-1694"
+        }
+    },
```

Comment:
Are these needed for anything?

Reply:
Yes, our build script needs these variables to have the official build branding in the About Dialog instead of showing only the git commit SHA.